Chess/BoardView.swift

```diff
@@ -171,6 +175,20 @@ class BoardView: UIView {
         updatePieces()
         updateMoveIndicators()
     }
+    
+    private func updateTheme() {
```

Comment:
Please let me know if there is an optimization for this function. I don't like inserting new view to the bottom of a view hierarchy. But couldn't find a better way.